ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -0,0 +1,174 @@
+    KCLI_PARAM="-P openshift_image=registry.ci.openshift.org/ocp/release:${PROW_JOB_ID/-telco5g/}"
+elif [ ! -z $JOB_NAME ]; then
+    # In case of regular periodic job
+    tmpvar="${JOB_NAME/*nightly-/}"
+    ocp_ver="${tmpvar/-e2e-telco5g/}"
+    KCLI_PARAM="-P openshift_image=registry.ci.openshift.org/ocp/release:$ocp_ver"
+fi
+echo "==========  Running with KCLI_PARAM=$KCLI_PARAM  =========="
+
+# Set environment for jobs to run
+INTERNAL=true
+INTERNAL_ONLY=true
+# Whether to use the bastion environment
+BASTION_ENV=true
+# Environment - US lab, DS lab or any
+CL_SEARCH="upstreambil"
+
+if $INTERNAL_ONLY; then
+    CL_SEARCH="internalbos"
+elif $INTERNAL; then
+    CL_SEARCH="any"
+fi
+
+cat << EOF > $SHARED_DIR/bastion_inventory
+[bastion]
+${BASTION_IP} ansible_ssh_user=centos ansible_ssh_common_args="$COMMON_SSH_ARGS" ansible_ssh_private_key_file="${SSH_PKEY}"
+EOF
+
+cat << EOF > $SHARED_DIR/get-cluster-name.yml
+---
+- name: Grab and run kcli to install openshift cluster
+  hosts: bastion
+  gather_facts: false
+  tasks:
+  - name: Discover cluster to run job
+    command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster_all.py --get-cluster -e $CL_SEARCH
+    register: cluster
+    environment:
+      JOB_NAME: ${JOB_NAME:-'unknown'}
+  - name: Create a file with cluster name
+    shell: echo "{{ cluster.stdout }}" > $SHARED_DIR/cluster_name
+    delegate_to: localhost
+EOF
+
+
+ansible-playbook -i $SHARED_DIR/bastion_inventory $SHARED_DIR/get-cluster-name.yml -vv
+# Get all required variables - cluster name, API IP, port, environment
+# shellcheck disable=SC2046,SC2034
+IFS=- read -r CLUSTER_NAME CLUSTER_API_IP CLUSTER_API_PORT CLUSTER_ENV <<< "$(cat ${SHARED_DIR}/cluster_name)"
+PLAN_NAME="${CLUSTER_NAME}_ci"
+
+cat << EOF > $SHARED_DIR/release-cluster.yml
+---
+- name: Release cluster $CLUSTER_NAME
+  hosts: bastion
+  gather_facts: false
+  tasks:
+
+  - name: Release cluster from job
+    command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster_all.py --release-cluster $CLUSTER_NAME
+EOF
+
+if [[ "$CLUSTER_ENV" != "upstreambil" ]]; then
+    BASTION_ENV=false
+fi
+
+if $BASTION_ENV; then
+# Run on upstream lab with bastion
+cat << EOF > $SHARED_DIR/inventory
+[hypervisor]
+${HYPERV_IP} ansible_host=${HYPERV_IP} ansible_user=kni ansible_ssh_private_key_file="${SSH_PKEY}" ansible_ssh_common_args='${COMMON_SSH_ARGS} -o ProxyCommand="ssh -i ${SSH_PKEY} ${COMMON_SSH_ARGS} -p 22 -W %h:%p -q centos@${BASTION_IP}"'
+EOF
+
+else
+# Run on downstream cnfdc1 without bastion
+cat << EOF > $SHARED_DIR/inventory
+[hypervisor]
+${DSHVIP} ansible_host=${DSHVIP} ansible_ssh_user=kni ansible_ssh_common_args="${COMMON_SSH_ARGS}" ansible_ssh_private_key_file="${SSH_PKEY}"
+EOF
+
+fi
+echo "#############################################################################..."
+echo "========  Deploying plan $PLAN_NAME on cluster $CLUSTER_NAME with$(if ! $BASTION_ENV; then echo "out"; fi) a bastion  ========"
```

Comment:
Can we change the if so it only prints "with bastion"? "without bastion" might be confusing.

Reply:
fixing in https://github.com/openshift/release/pull/32537